src/main/java/org/swisspush/reststorage/util/ModuleConfiguration.java

```diff
@@ -232,6 +244,14 @@ public int getRedisPort() {
         return redisPort;
     }
 
+    public int getRedisReconnectAttempts() {
+        return redisReconnectAttempts;
+    }
+
+    public int getRedisReconnectDelaySec() {
+        return Math.max(redisReconnectDelaySec, 1);
```

Comment:
```java
    Math.max(redisReconnectDelaySec, 1)
```
Should we instead throw in the setter on bad values?

Reply:
The mapping from JSON to the POJO
```java
public static ModuleConfiguration fromJsonObject(JsonObject json) {
	ModuleConfiguration mc = json.mapTo(ModuleConfiguration.class);
	return mc;
}
```
does not use the setter methods. Invalid values coming from a JsonObject are just used.

Reply:
Ah I see :D

Then I think we at least should log it to save hair from getting pulled out:
```java
    if( redisReconnectDelaySec < 1 ){
        LOG.debug("Will ignore redisReconnectDelay {}s (too small) and use 1 instead", redisReconnectDelaySec);
        return 1;
    }
    return redisReconnectDelaySec;
```